src/Service/SearchEngineManager.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\Skill;
+use App\Repository\SkillRepository;
+
+use App\Entity\Usergroup;
+use App\Repository\UsergroupRepository;
+
+use App\Entity\User;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Container\ContainerInterface;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\FormType;
+use Symfony\Component\Form\Extension\Core\Type\SearchType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+use Symfony\Component\Intl\Intl;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+use App\Form\TagType;
+
+class SearchEngineManager {
+	private $manager;
+	private $formFactory;
+
+	public function __construct ( EntityManagerInterface $manager, ContainerInterface $container ) {
+		$this->manager     = $manager;
+		$this->formFactory = $container->get( 'form.factory' );
+	}
+
+
+	public function getForm ( $filters, $options = [] ) {
+
+		$tagArray = array_combine($filters[ 'keywords' ], $filters[ 'keywords' ]);
+
+		$form = $this->formFactory	->createBuilder( FormType::class, $filters )
+                                  	->setMethod( 'get' )
+									->add( 'resultType', ChoiceType::class, [
+											'required' => FALSE,
```

Comment:
```suggestion
											'required' => false,
```
Utilise plutôt les booléens en minuscules. En php les majuscules sont pour les constantes. Ici ce n'est pas la constante qui nous intéresse, mais la valeur.